relay-redis/src/config.rs

```diff
@@ -5,6 +5,10 @@ use serde::{Deserialize, Serialize};
 pub struct RedisConfigOptions {
     /// Maximum number of connections managed by the pool.
     pub max_connections: u32,
+    /// Minimum amount of idle connections kept alive in the pool.
+    ///
+    /// If not set it will default to [`Self::max_connections`].
```

Comment:
```suggestion
    /// If not set it will default to a fifth of [`Self::max_connections`] (rounded up).
```

Reply:
If this value is `None` we end up passing `None` to the pool, which will default to using `max_connections`. We only have the 'special' logic in the `config` crate. Maybe that's not ideal 🤔